tests/bearlib/languages/documentation/DocumentationExtractionTest.py

```diff
@@ -35,32 +35,32 @@ def test_extract_documentation_C(self):
         docstyle_C_doxygen = DocstyleDefinition.load("C", "doxygen")
 
         expected_results = (DocumentationComment(
-                                ("\n"
-                                 " This is the main function.\n"
-                                 "\n"
-                                 " @returns Your favorite number.\n"),
-                                docstyle_C_doxygen.markers[0],
-                                TextRange.from_values(3, 1, 7, 4)),
-                            DocumentationComment(
+            ("\n"
+             " This is the main function.\n"
+             "\n"
+             " @returns Your favorite number.\n"),
+            docstyle_C_doxygen.markers[0],
+            TextRange.from_values(3, 1, 7, 4)),
+            DocumentationComment(
                                 ("\n"
                                  " Preserves alignment\n"
                                  " - Main item\n"
                                  "   - sub item\n"
                                  "     - sub sub item\n"),
-                                docstyle_C_doxygen.markers[2],
-                                TextRange.from_values(15, 1, 20, 4)),
-                            DocumentationComment(
+            docstyle_C_doxygen.markers[2],
+            TextRange.from_values(15, 1, 20, 4)),
+            DocumentationComment(
                                 (" ABC\n"
                                  "    Another type of comment\n"
                                  "\n"
                                  "    ..."),
-                                docstyle_C_doxygen.markers[1],
```

Comment:
the stuff above then doesn't look so PEP8 conform... at least it looks quite ugly having the first param way more indented then the following ones^^